Makefile

```diff
@@ -30,6 +30,9 @@ ifeq ($(TESTS), true)
 	ENDPOINT_CFLAGS += -C -DENABLE_ENDPOINT_TEST
 endif
 
+# Pass target for endpoint build process
+ENDPOINT_CFLAGS += -C -DTARGET=$(TARGET)
```

Comment:
Why `-C`?

Reply:
This is the `mkapp` flags for passing cflags into compiler. The `mkapp` is the tool to build endpoint app provides by the Sierra.

Reply:
See https://docs.legato.io/latest/buildToolsmkapp.html

Reply:
`-C` is not a typical gcc option. Move out of this setting.

Reply:
The `-C` is the flag for `mkapp` to specify extra flags to be passed to the C compiler.

Reply:
Then, let's call it `LEGATO_FLAGS`. Don't confuse.